### DISCUSSION TITLE
train my IOB format using spacy

### DISCUSSION THREAD

**Discussion by myeghaneh** [explosion/spaCy#6724] [Help: Coding & Implementations] [answered]
I have data in this format, (IOB format on token level)

```
[On, IN, O], [the, DT, O], [other, JJ, O], [hand, NN, O], [,, ,, O], [the, DT, B-P], [significance, NN, I-P], [of, IN, I-P], [competition, NN, I-P], [is, VBZ, I-P], [that, IN, I-P], [how, WRB, I-P], [to, TO, I-P], [become, VB, I-P], [more, JJR, I-P], [excellence, NN, I-P], [to, TO, I-P], [gain, VB, I-P], [the, DT, I-P], [victory, NN, I-P], [., ., O], [Hence, RB, O], [it, PRP, O], [is, VBZ, O], [always, RB, O], [said, VBN, O], [that, IN, O], [competition, NN, B-C], [makes, VBZ, I-C], [the, DT, I-C], [society, NN, I-C], [more, RBR, I-C], [effective, JJ, I-C], [., ., O],

````
how can I train a classifer which work like NER by spacy?  (My case is not NER , but the format is close)

should I convert my data to spacy format?

**Comment by aakashb95** [comment]
Any luck with this?

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @myeghaneh , if you have data in the IOB format, perhaps you can check out the spaCy [`convert`](https://spacy.io/api/cli#convert) command? You can refer [to these examples](https://github.com/explosion/spaCy/tree/master/extra/example_data/ner_example_data) for more info. It might even make sense to save them as an `.iob` file first before running the command.

Once you have the `.spacy` files, you can now [start training](https://spacy.io/usage/training).
